src/vs/workbench/services/configurationResolver/common/variableResolver.ts

```diff
@@ -38,11 +40,13 @@ export class AbstractVariableResolverService implements IConfigurationResolverSe
 	private _context: IVariableResolveContext;
 	private _labelService?: ILabelService;
 	private _envVariablesPromise?: Promise<IProcessEnvironment>;
+	private _pathService: IPathService | undefined;
 	protected _contributedVariables: Map<string, () => Promise<string | undefined>> = new Map();
 
-	constructor(_context: IVariableResolveContext, _labelService?: ILabelService, _envVariablesPromise?: Promise<IProcessEnvironment>) {
+	constructor(_context: IVariableResolveContext, _labelService?: ILabelService, _envVariablesPromise?: Promise<IProcessEnvironment>, _pathService?: IPathService) {
```

Comment:
nit: can `_pathService` move to be after `_labelService` so that the services are together.

Reply:
Makes sense. Fixed.